modules/monitoring-setting-the-body-size-limit-for-metrics-scraping.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+:_content-type: PROCEDURE
+[id="setting-the-body-size-limit-for-metrics-scraping_{context}"]
+= Setting the body size limit for metrics scraping
+
+By default, no limit exists for the uncompressed body size for data returned from scraped metrics targets.
+You can set a body size limit to help avoid situations in which Prometheus consumes excessive amounts of memory when scraped targets return a response that contains a large amount of data.
+In addition, by setting a body size limit, you can reduce the impact that a malicious target could have on Prometheus and on the cluster as a whole.
```

Comment:
```suggestion
In addition, by setting a body size limit, you can reduce the impact that a malicious target can have on Prometheus and on the cluster as a whole.
```
IBM SG: Use [can] to indicate ability. Use [can] instead of "could". To indicate possibility, use "might". To indicate permission, use "may". In most cases, use "can" or "might", not "may".

Reply:
Thanks. Will change to `might`